staging/src/k8s.io/component-helpers/scheduling/corev1/helpers_test.go

```diff
@@ -545,3 +546,41 @@ func TestMatchNodeSelectorTermsStateless(t *testing.T) {
 		})
 	}
 }
+
+func TestIsNativeResource(t *testing.T) {
```

Comment:
Could you add tests for the other functions?

Perhaps I would have a single test table:

```
[]struct {
  resourceName v1.ResourceName
  isNative bool
  isExtended
  ...
```

But separate is ok too.

Reply:
Yeah, I will add missing tests for each resource name helper.

Do you have any real examples of resource names in `kubernetes.io` domain? I have not been able to find any so far. Are there any beside the implicit ones (like `cpu`, `memory`, `pods`)? E.g. `kubernetes.io/inodes`.

Reply:
`ephemeral-storage` https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container

Reply:
Missing test cases provided